source/_components/sensor.trafikverket_weatherstation.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: page
+title: "Trafikverket WeatherStation"
+description: "Instructions how to integrate Trafikverket WeatherStation within Home Assistant."
+date: 2018-02-01 12:06
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: trafikverket.png
+ha_category: Sensor
+ha_release: 0.63.0
+---
+
+Showing weather information for air and road temperature provided by Trafikverket in Sweden. 
+
+Usage: Get weather data if you live near a weather station, just want to have the current temperature, needing the current temperature to know when your car heating system should start or when to leave for work due to queues (because the road is frozen).
```

Comment:
This needs rephrasing, currently it's very unclear languagewise. Maybe have a list `Potential use-cases`?

Reply:
Updated, please review.